client/src/pages/home/index.tsx

```diff
@@ -0,0 +1,285 @@
+import { startsWith, toLower, toString } from 'lodash';
+import React, { FC, useEffect, useState } from 'react';
+import {
+  DropdownItemProps,
+  Input,
+  Menu,
+  Segment,
+  Select,
+} from 'semantic-ui-react';
+import { IIssue, IPitch, IUserAggregate } from 'ssw-common';
+
+import { getAggregatedUser, isError } from '../../api';
+import {
+  InterestsSelect,
+  SubmitPitchModal,
+  Walkthrough,
+} from '../../components';
+import DynamicTable from '../../components/Tables/DynamicTable';
+import { Sort, View } from '../../components/Tables/DynamicTable/types';
+import { useAuth } from '../../contexts';
+import { pagesEnum, pitchStatusEnum } from '../../utils/enums';
+import { filterPitchesByInterests, titleCase } from '../../utils/helpers';
+
+import {
+  filterCreatedYear,
+  filterRequestClaimYear,
+  filterStatus,
+  getRecordsForTab,
+  getSearchFields,
+  getYearsSinceSSWEstablished,
+  isPitchArray,
+  Tab,
+  TABS,
+} from './helpers';
+import './styles.scss';
+import { getColumnsForTab } from './views';
+
+const Homepage: FC = () => {
+  type RecordType = IPitch | IIssue;
+
+  const { user } = useAuth();
+
+  const [refreshRecords, setRefreshRecords] = useState<boolean>(false);
+  const [aggregatedUser, setAggregatedUser] = useState<IUserAggregate>();
+
+  const [currentTab, setCurrentTab] = useState<Tab>(TABS.MEMBER_PITCHES);
+
+  const [searchInput, setSearchInput] = useState<string>('');
+  const [filteredInterests, setFilteredInterests] = useState<string[]>([]);
```

Comment:
This one DOES need the string array typing so it knows what kind of array it is.